oldschnorr.go

```diff
@@ -0,0 +1,161 @@
+package secp256k1
+
+// #include "./depend/secp256k1/include/secp256k1_oldschnorr.h"
+import "C"
+import (
+	"encoding/hex"
+	"github.com/pkg/errors"
+	"unsafe"
+)
+
+// SchnorrPublicKey is a PublicKey type used to sign and verify Schnorr signatures.
+// The struct itself is an opaque data type that should only be created via the supplied methods.
+type SchnorrPublicKey struct {
+	pubkey C.secp256k1_pubkey
+}
+
+// SchnorrSignature is a type representing a Schnorr Signature.
+// The struct itself is an opaque data type that should only be created via the supplied methods.
+type SchnorrSignature struct {
+	signature [64]byte
+}
+
+// String returns the SchnorrSignature as the hexadecimal string
+func (signature *SchnorrSignature) String() string {
+	return hex.EncodeToString(signature.signature[:])
+}
+
+// Serialize returns a 64 byte serialized signature
+func (signature *SchnorrSignature) Serialize() [64]byte {
+	return signature.signature
+}
+
+// DeserializeSchnorrSignature deserializes a 64 byte serialized schnorr signature into a SchnorrSignature type.
+func DeserializeSchnorrSignature(serializedSignature [64]byte) *SchnorrSignature {
+	return &SchnorrSignature{signature: serializedSignature}
+}
+
+// String returns the SchnorrPublicKey as the hexadecimal string
+func (key *SchnorrPublicKey) String() string {
+	return hex.EncodeToString(C.GoBytes(unsafe.Pointer(&key.pubkey.data[0]), C.int(len(key.pubkey.data))))
```

Comment:
More details on this: 
https://github.com/golang/go/issues/38002
https://daglabs.atlassian.net/browse/NOD-856

Reply:
Got rid of this